src/app/components/SpeakingDog/SpeakingDog.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import SpeechBubble from "../Icons/SpeechBubble";
+import LogoIcon from "../Icons/logoIcon";
+import styles from "./SpeakingDog.module.css";
+
+type SpeakingDogProps = {
+  speech: string;
+};
+
+export default function SpeakingDog({ speech }: SpeakingDogProps): JSX.Element {
+  return (
+    <>
+      <div className={styles.bubble}>
+        <SpeechBubble speech={speech} />
+      </div>
+      <div className={styles.dog}>
+        <LogoIcon />
+      </div>
+    </>
```

Comment:
```suggestion

      <div className={styles.bubble}>
        <SpeechBubble speech={speech} />
      </div>
      <div className={styles.dog}>
        <LogoIcon />
      </div>

```

Reply:
When I delete the fragments I get a "JSX expressions must have one parent element.ts(2657)" error so I will keep them.